test/e2e/utils/cleanup.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"context"
+
+	"sigs.k8s.io/cluster-api-provider-azure/test/e2e/auth"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
+	"github.com/Azure/go-autorest/autorest"
+	azureAuth "github.com/Azure/go-autorest/autorest/azure/auth"
+)
+
+// GetResource gets the resource group if exist
+func GetResource(ctx context.Context, creds auth.Creds, resourceName string) (resources.Group, error) {
+	resource, err := getResource(ctx, creds, resourceName)
+	return resource, err
+}
+
+// Cleanup deletes the resource group created during the testing
+func Cleanup(ctx context.Context, creds auth.Creds, resourceName string) error {
+	_, err := deleteResource(ctx, creds, resourceName)
```

Comment:
are you dropping the future on purpose here? If not, let's not drop it. If so, we should be more explicit about not waiting for the long running operation (maybe with a comment?)

Reply:
ok, let me check that

Reply:
done, i think i did right

Reply:
I did locally a test and it worked